pep_sphinx_extensions/pep_processor/transforms/pep_headers.py

```diff
@@ -204,7 +204,7 @@ def _process_pretty_url(url: str) -> tuple[str, str]:
         item_name, item_type = LINK_PRETTIFIERS[parts[2]](parts)
     except KeyError as error:
         raise ValueError(
-            "{url} not a link to a recognized domain to prettify") from error
+            f"{url} not a link to a recognized domain to prettify") from error
```

Comment:
Thanks, that's my fault there...though not sure who reviewed that PR :stuck_out_tongue:

Reply:
Let's blame it on the lack of tests ;)